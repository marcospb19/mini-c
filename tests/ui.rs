#![feature(custom_inner_attributes)]
#![rustfmt::skip]

use sushi::test_sushi;
use unindent::unindent;

#[test]
fn empty_statements() {
    let input = unindent("
        ;
        ;;
    ");
    insta::assert_debug_snapshot!(test_sushi(input));
}

#[test]
fn empty_program() {
    let input = "";
    insta::assert_debug_snapshot!(test_sushi(input));
}
