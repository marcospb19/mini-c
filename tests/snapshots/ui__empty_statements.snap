---
source: tests/ui.rs
expression: test_sushi(input)
---
Ok(
    OutputCapture {
        stdout: "",
        stderr: "",
    },
)
